extension/data/deinflect.dat

```diff
@@ -333,4 +336,11 @@ imperative negative
 り	る	640	24
 れ	る	640	23
 れ	れる	384	24
-ろ	る	384	23
\ No newline at end of file
+ろ	る	384	23
+した	す	8320	14
```

Comment:
```suggestion
でした	だ	8320	6
```

Reply:
I think if we map like this (similar to how most verbs and adjectives are mapped) then we shouldn't need to make additional changes outside the new mappings and we can add as many or few as we want to fill the holes you want!

Reply:
I see. So instead of conjugating でした to です, I should conjugate it to だ? And also do this for every instances of だ?
What about inflection of でございます?　Should I considerでございます as the base form as I believe でござる is not commonly used.

Reply:
Or should I deinflect でございます back to だ and add a new 'honorific' category in deinflect.dat?

Reply:
Yes, you understood it well. This is definitely a special case and there's room for iteration.

One interesting idea from @Stephie was to add mappings to the both だ and です so that people could see both connections. (and similarly maybe have honorific forms go to both だ and でございます)

I think honorific forms usually are treated separately so it's I think it's a fair point to leave it as is. I also agree that deinflecting it to でございます makes sense as the plain form is archaic.

To summarize the simplest approach I think we should take for now:
- です forms go to だ
- でございます forms go to でございます

One thing to note is since these are irregular inflections you should map the entire word instead of just the suffix like you've done till now. (simmilar to how する is treated I think) 
There's no reason to be generic since each line only captures a single mapping as far as I can tell.

Reply:
So, I tried deinflecting to だ but when I tested it, the words are once again not recognized. I feel like it has something to do with this line of code in the deinflection implementation.
https://github.com/melink14/rikaikun/blob/69c7fe799955df76c94e9686ea08ad0b8503e9ae/extension/data.ts#L281
Am I right? If this is really the case, then we can not use だ as the base form to deinflect to since the length is always going to be 1, is that the case?

Reply:
I went with option 1 and did not find any instances where rikaikun produces any weird output yet. I think I would prefer going with option 1 for now.